codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/PresignerGenerator.kt

```diff
@@ -50,6 +51,16 @@ class PresignerGenerator : SwiftIntegration {
                 var serviceConfig = AWSServiceConfig(writer, protoCtx)
                 renderPresigner(writer, ctx, delegator, op, inputType, serviceConfig)
             }
+            // Expose presign-request as a method for service client object
+            val symbol = protoCtx.symbolProvider.toSymbol(protoCtx.service)
+            protoCtx.delegator.useFileWriter("./${ctx.settings.moduleName}/${symbol.name}.swift") { writer ->
+                renderPresignAPIInServiceClient(writer, symbol.name, op, inputType)
+                // Add special-case import statement for Linux
+                writer.write("// In Linux, Foundation.URLRequest is moved to FoundationNetworking.")
```

Comment:
I would not render a comment like this into generated code.  Instead, it can be a comment here in the codegen source file.

Reply:
Changed as suggested.